### PR TITLE
Template update

### LINKED ISSUE #2: Update pipeline template
### Description of feature

Update to the latest nf-core pipeline template

### PR DESCRIPTION
**Pull request by muffato**
Closes #2.

Here I've updated the pipeline template from 2.5 to 2.8, one version at a time.

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/sanger-tol/sequencecomposition/tree/master/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by muffato** [comment]
The CI fails when doing the "full" test because there isn't enough space on disk 😁 The regular test does work already.

I'll try to reduce the disk usage, probably by trimming the samplesheet.

**Comment by muffato** [comment]
Added a few commits to address issues actually found in https://github.com/sanger-tol/ensemblgenedownload/pull/6 and a commit to enable the `cleanup` profile for our Tower tests and close #9

File: .github/workflows/sangerfulltest.yml
Comment (priyanka-surana) [inline .github/workflows/sangerfulltest.yml:1]: Can we rename this to match nf-core `test_full` – `test_full_sanger`?

File: .github/workflows/sangertest.yml
Comment (priyanka-surana) [inline .github/workflows/sangertest.yml:1]: Same as above

File: .nf-core.yml
Comment (priyanka-surana) [inline .nf-core.yml:1]: Can you add Code of Conduct document back and remove here?

File: CHANGELOG.md
Comment (priyanka-surana) [inline CHANGELOG.md:1]: Can you fix the release version, name and date for v1? Also add information for the next release.

**Review by priyanka-surana — CHANGES_REQUESTED** [review]
Almost there, a few changes to documentation.

**Review by priyanka-surana — APPROVED** [review]
LGTM 👍🏼 Thanks for making the changes

**Pull request merged**